app/views/apis/new.html.erb

```diff
@@ -2,12 +2,10 @@
 
 <p>
   <%= form_for @api do |f| %>
-    <%= f.label :name %>
-    <%= f.text_field :name %>
-    <%= f.label :url, "URL" %>
-    <%= f.text_field :url %>
-    <%= f.label :description %>
-    <%= f.text_field :description %>
+    <%= f.text_field :name, id: "Name" %>
+    <%= f.text_field :url, id: "URL" %>
+    <%= f.text_field :description, id: "Description" %>
+    <%= f.text_field :tags, value: "", id: "Tags" %>
```

Comment:
Removed the labels for the fields? This is typically how capybara finds these fields. I would opt to hide the labels using CSS, instead: http://www.coolfields.co.uk/2011/04/accessible-forms-should-every-input-have-a-label/